lib/App/Sqitch/Engine.pm

```diff
@@ -269,6 +274,15 @@ sub deploy {
     $self->$meth( $plan, $to_index );
 }
 
+# Do a thing similar to Sqitch::Plan::Change::format_name_with_tags,
```

Comment:
If you can't use that actual method, I suggest adding a new one there, not here.

Reply:
We do not have a Sqitch::Plan::Change here, so I don't think this code can go in that module?

Reply:
Oh 🤦🏻 got it.

Reply:
I do wonder if we should avoid the overhead when verbosity is less than two. Though it probably isn't that expensive, relative to the overhead of the database. Maybe I'm overthinking it.